tests/test_logging.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import logging
+import os
+from tempfile import TemporaryDirectory
+from pkg_resources import resource_filename
+from wps_tools.utils import log_handler
+from .common import Response
+from .processes.wps_test_process import TestProcess
+
+logger = logging.getLogger()
+
+formatter = logging.Formatter(
+    "%(asctime)s %(levelname)s: wps-tools: %(message)s", "%Y-%m-%d %H:%M:%S"
+)
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+@pytest.mark.parametrize(("message"), ["Process completed"])
+@pytest.mark.parametrize(("process_step"), ["complete"])
+@pytest.mark.parametrize(("level"), ["INFO"])
```

Comment:
Generally if you parametrize items in this way it is because you want to run the items with every combination. In this case we just have one combination, we can probably put this into a single decorator.